.github/workflows/blackbox-lts.yml

```diff
@@ -0,0 +1,74 @@
+name: Blackbox LTS Tests
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - api/**
+      - tests/blackbox/**
+      - packages/**
+      - package.json
+      - pnpm-lock.yaml
+      - .github/workflows/blackbox-lts.yml
+  workflow_call:
+
+concurrency:
+  group: blackbox-lts-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  NODE_OPTIONS: --max_old_space_size=6144
+
+jobs:
+  db:
+    name: ${{ matrix.vendor }}
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+    strategy:
+      fail-fast: false
+      matrix:
+        vendor:
+          - postgres-lts-16
+          - postgres-lts-15
+          - postgres-lts-14
+          - postgres-lts-13
+          - mysql-lts-8
+          - maria-lts-10_11
+          # - maria-lts-10_6
```

Comment:
Intentionally left out. `mariadb 10.6` specifically has an issue where `uploaded_on` is set to non nullable when it is expected to be nullable. This results in our `file` tests failing.